pkg/cmd/cli/cmd/rollback.go

```diff
@@ -57,14 +60,14 @@ func NewCmdRollback(fullName string, f *clientcmd.Factory, out io.Writer) *cobra
 	}
 
 	cmd := &cobra.Command{
-		Use:     "rollback DEPLOYMENT",
+		Use:     "rollback [dc/]config-name",
```

Comment:
Add new flags, deprecate old ones, or allow a wider range of options.  But
we must manage cli syntax change exactly like API or config change -
slowly, with explicit deprecation and management.

On Jul 9, 2015, at 12:52 PM, Dan Mace <notifications@github.com> wrote:

In pkg/cmd/cli/cmd/rollback.go
<https://github.com/openshift/origin/pull/3643#discussion_r34276449>:

> @@ -57,14 +60,14 @@ func NewCmdRollback(fullName string, f *clientcmd.Factory, out io.Writer) *cobra
>  	}
>
>  	cmd := &cobra.Command{
> -		Use:     "rollback DEPLOYMENT",
> +		Use:     "rollback [dc/]config-name",

We can never change the syntax of this command? What is the proposed syntax
that gets us to a place where we can accept a config? rollback
--to-config=config --to-version=N (no arg[0]?)

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/3643/files#r34276449>.